docs/app/docs/jetify_cloud/dashboard/inviting_members.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Inviting your Team Members
+sidebar_position: 3
+---
+
+If you have a paid Team project, you can invite team members to join your project. Team members can configure and use your project’s secrets when developing using the Devbox or Envsec CLI.
+
+:::info
+If you want to invite team members to your projects, you will need to add a payment option and upgrade your account to a Team Account. 
+:::
+
+Team members can be added in one of two Roles:
+
+- **Member:** Members can access or add projects, as well as access or modify the secrets for their projects
+- **Admins:** Have all the permissions of Members, but can also invite new team members and configure project billing
+
+## Adding a Team Member
+
+1. In the Jetify Dashboard, navigate to the **Members** tab
+
+   ![Members Tab](../../../static/img/members_tab.png)
+
+2 . To invite a member, enter their email address, and the **Role** you want to assign to them.
+
+![Inviting a Member](../../../static/img/inviting_member.png)
+
+1. Once invited, users will receive an email with instructions on how to join your team. They will show up in the **Members** tab as invited until the accept the invitation:
+
+   ![Invited Member](../../../static/img/invited_member.png)
+
+   Once they accept the invitation, they will switch to **active**
+
+   ![Active Member](../../../static/img/active_member.png)
```

Comment:
Should we mention that the invitation expires in 7 days? And you have to re-invite them